From 8bb2d284bbc326a0f4cd8f1bcec3abdbb88d9572 Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@st.com>
Date: Wed, 1 Feb 2023 17:05:48 +0100
Subject: [PATCH 410/529] ARM: configs: enable Ethernet REALTEK Phy on
 fragment-01-multiv7_cleanup

Need to use Realtek PHY driver instead of GENERIC Phy driver by default
If Generic Phy used, issue to retrieve IP address after suspend/resume

Signed-off-by: Christophe Roullier <christophe.roullier@st.com>
Change-Id: I6ab1e4a80c19107e9a350a6c9fdf83623476e8b4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/288507
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Alexandre TORGUE <alexandre.torgue@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/290455
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm/configs/fragment-01-multiv7_cleanup.config | 1 -
 1 file changed, 1 deletion(-)

--- a/arch/arm/configs/fragment-01-multiv7_cleanup.config
+++ b/arch/arm/configs/fragment-01-multiv7_cleanup.config
@@ -74,7 +74,6 @@
 # CONFIG_POWER_AVS is not set
 # CONFIG_POWER_SUPPLY is not set
 # CONFIG_PREEMPT_NONE is not set
-# CONFIG_REALTEK_PHY is not set
 # CONFIG_REGULATOR_WM8994 is not set
 # CONFIG_SCHED_DEBUG is not set
 # CONFIG_SECTION_MISMATCH_WARN_ONLY is not set
